lib/webengine.h

```diff
@@ -0,0 +1,31 @@
+/****************************************************************************
+**
+** Copyright (C) 2016 Jolla Ltd.
+** Contact: Raine Makelainen <raine.makelaine@jolla.com>
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef SAILFISHOS_WEBENGINE_H
+#define SAILFISHOS_WEBENGINE_H
+
+#include <QString>
+#include <qmozcontext.h>
+
+namespace SailfishOS {
+
+class WebEngine : public QMozContext {
```

Comment:
Are we trying to get rid of QMozContext altogether at some stage?  Or not?  Just wondering if it makes sense to extend the public WebEngine class here, or instead provide some API and internally call through to the appropriate function of a QMozContext private data instance...

This is probably fine as-is, just thought I'd ask.